oaknational/Oak-Web-Application
# Feat/pupil 1129/teacher expiring banner

### Job Story

**When** I navigate to a lesson or lesson listing which is due to be expired
**I want to see the new lesson takedown banner**
**So that users can plan appropriately**.

### Implementation details

The banner appears in response to the action “display_expiring_banner”

https://www.figma.com/design/dGO9erL0b0kpYxmv6Jku4f/Lessons-takedown-banners?node-id=2011-10285&m=dev

### Acceptance criteria

- [x]  banner matches design
- [x]  link goes to appropriate place
- [x]  teacher lesson page
- [x]  teacher lesson canonical page
- [x]  teacher lesson listing

## How to test

Currently no expired lessons are available on production for testing

1. Go to https://deploy-preview-3058--oak-web-application.netlify.thenational.academy

VIsit legacy unitlistings, lesson listings and lessons for any of  

--   Legacy content to display expiry banner
--   ks1: ["english", "geography", "history", "science", "maths"],
--   ks2: [
--     "english",
--     "english-grammar",
--     "english-reading-for-pleasure",
--     "english-spelling",
--     "geography",
--     "history",
--     "science",
--     "maths",
--   ],
--   ks3: ["english", "history", "science", "maths"],
--   ks4: ["biology", "chemistry", "combined-science", "maths"]

Click on the view new resources buttons and check that the link goes to the unit listing or programmes page appropriately.
